ui/src/pages/ManualTaskPage.vue

```diff
@@ -235,12 +236,15 @@ const taskColumns = ref([
   }
 ])
 
+// Computed data
+const stageOptions = computed(() => releaseStages.stages)
```

Comment:
I realize JavaScript has some peculiarities in how variables are defined, but would it be more clear to move this below the declaration of `releaseStages`?

Reply:
Done, yes, for readability this makes sense